.github/workflows/periodic_benchmarks.yml

```diff
@@ -44,7 +44,8 @@ jobs:
           asv machine --machine "GitHubRunner"
           asv run --machine "GitHubRunner" NEW --show-stderr -v
         env:
-          SUNDIALS_INST: $HOME/.local
+          # TODO: test this
+          SUNDIALS_INST: sundials_KLU_libs/
           LD_LIBRARY_PATH: $HOME/.local/lib
```

Comment:
```suggestion
```

Both of these shouldn't be needed anymore. You'll need to update the build command in `asv.conf.json`, though, it still uses `setup.py`. Once you manage to get the benchmarks working, let me know with a link to the logs :)